renku/cli/_graph.py

```diff
@@ -233,7 +233,9 @@ def dependencies(self, revision='HEAD', paths=None):
         """Return dependencies from a revision or paths."""
         result = []
 
-        if not paths:
+        if paths:
+            paths = map(lambda path: self.normalize_path(path), paths)
```

Comment:
```suggestion
            paths = [self.normalize_path(p) for p in paths]
```

Reply:
Thanks for the suggestion @jsam - I agree that would be a bit nicer to read. I used `map` instead of a list comprehension because `paths` can be a generator and the list comprehension would force the paths to be evaluated at this point, which - as far as I know - is not the case when using `map`.

Reply:
You can use a generator expression

Reply:
> You can use a generator expression

ok, done